pandas/tests/copy_view/test_array.py

```diff
@@ -132,21 +132,25 @@ def test_series_array_ea_dtypes(using_copy_on_write):
         assert arr.flags.writeable is True
 
     arr = np.asarray(ser)
-    assert not np.shares_memory(arr, get_array(ser))
-    assert arr.flags.writeable is True
+    assert np.shares_memory(arr, get_array(ser))
```

Comment:
Are these related changes?

Reply:
Yes, previously we roundtripped through object which triggered a copy, this is a view now! (that's an improvement)